### DISCUSSION TITLE
Suggestion

### DISCUSSION THREAD

**Discussion by Marco-9456** [JunkFood02/Seal#76] [General]
Add Planned and Rejected stuff to README so that people don't duplicate the same issues and suggestions
and maybe a Discord server so that users can share there Custom Commands and help with the project.

**Comment by JunkFood02** [comment]
Thanks for your suggestions! I didn't consider that so many people would be interested in this small project. 😳

**Comment by s4velyeva** [comment]
That was obvious, not all yt-dlp wrappers for android are so useful and cute :)
I'm already making ukrainian locale, so it will fit for our users :3

**Comment by shuvashish76** [comment]
> maybe a Discord server

Matrix would be better IMO. Matrix community growing fast recently [crossed 60 million users](https://news.itsfoss.com/matrix-sixty-million-users/).

**Reply by s4velyeva** [reply]
Discord will become handy to make any community events, Matrix will be good for community messaging, and Telegram will be good solution for any announcements.

My boyfriend and some Telegram channels, russian and Ukrainian, already made posts about Seal, so community will now grow faster, but most bigger part of the community, i guess, will be ukrainian and russian.

**Comment by Marco-9456** [comment]
> 

Never heard of it

**Comment by shuvashish76** [comment]
@JunkFood02 This is a general discussion & not app specific issue... Maybe turn on "Discussion" option for this repo & convert this issue to a discussion.

**Comment by JunkFood02** [comment]
The social software I use more often is Telegram, so I prefer a telegram channel. 🤔
